pom.xml

```diff
@@ -1015,6 +1017,11 @@
               <artifactId>asm</artifactId>
               <version>${asm.version}</version>
             </dependency>
+            <dependency>
```

Comment:
I got explanation offline, but pls do share here the reason for adding this dependency for `maven-shade-plugin`